pkgs/by-name/sp/spirv-llvm-translator/package.nix

```diff
@@ -48,10 +48,10 @@ let
         hash = "sha256-OsDohXRxovtEXaWiRGp8gJ0dXmoALyO+ZimeSO8aPVI=";
       }
     else if llvmMajor == "14" then
-      {
-        version = "14.0.0+unstable-2024-07-15";
-        rev = "2823e7052b7999c10fff63bc8089e5aa205716f4";
-        hash = "sha256-8/4B74hYge6WiH7PzRGEgE3W7f9IkQ4VMmfkWKYA/l4=";
+      rec {
+        version = "14.0.10";
```

Comment:
Intel is using the tip of the llvm 14 branch usually. Did that change?

Reply:
I prefer to have a stable tag. I've tested with this one and it works.

Reply:
We can't do that. Intel has specific patches on that branch which are more or less required for this to work correctly. We cannot just dismiss them.

Reply:
The 14.0.10 release contains the tag that is references here: https://github.com/intel/intel-graphics-compiler/releases/tag/v2.8.3

Reply:
I am not sure if the extra fixes would break something. That part was especially hairy in the past 😅 

I usually take a look at and see how far it is off. Right now it is not that bad but sometimes it is like 20 commits and I myself cannot asset if any of those commits are relevant for the intel tools. :/
https://github.com/KhronosGroup/SPIRV-LLVM-Translator/compare/llvm_release_140